### ISSUE TITLE
FXI---scan_id is not consistent if running at two computers

### ISSUE THREAD

**Issue by gmysage** [NSLS-II/Bug-Reports#189]
Dear all, when I run `RE(scan(...))` at **xf18id-ws2**, the scan_id is starting at e.g. _1632_. However, if I run `RE(scan(...))` at another computer, saying at **xf18id-ws1**, it gives me the scan_id _580_. 
The problem is that, the old scan which was assigned to a scan_id of 580 has been **overwritten** when I retrieve the data using `h=db[580]`.

**Comment by tacaswell** [comment]
Yes, that is why it is printed out as 'transient scan ID'.   Use the uid instead which will be unique between sessions.
